packages/app/next.config.js

```diff
@@ -3,6 +3,8 @@ const nextConfig = {
   output: "export",
   distDir: "../../dist",
   webpack(config) {
+    config.resolve.fallback = { "pino-pretty": false, lokijs: false };
```

Comment:
why do we need this config?

Reply:
This libs were throwing some warning in the dev server. Found this solution to prevent fix it.